lib/rumTemplate.js.handlebars

```diff
@@ -1,8 +1,8 @@
 (function() {
 	var beacon = new Image();
-	beacon.src = '//{{{host}}}/v2/recordRumData?X';
+	beacon.src = '//{{{host}}}/v2/recordRumData?X' + 
```

Comment:
Confirmed:

http://qa.polyfill.io/v2/polyfill.js?rum=1

```
(function(undefined) {
(function() {
	var beacon = new Image();
	beacon.src = '//qa.polyfill.io/v2/recordRumData?X';
	"&Array.from=" + ...
```

So we will need to fix that in a patch release before deploying 3.13 to prod.  I'll make a separate PR for that since this one is obviously not likely to make it for 3.13. @JakeChampion